dash/frontend/src/app/core/enum/CVETypes.ts

```diff
@@ -0,0 +1,16 @@
+
+export enum CVETypes {
```

Comment:
We have an enum called VaulnerabilitySeverity that has the same content, but with the values capitalized. Is this doing something different enough to have both?

Reply:
Refactored to use that enum; put the abbreviations in that file too